packages/docs-infra/package.json

```diff
@@ -1,8 +1,18 @@
 {
   "name": "@mui/internal-docs-infra",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "author": "MUI Team",
   "description": "MUI Infra - internal documentation creation tools.",
+  "main": "./index.js",
+  "types": "./index.d.ts",
+  "exports": {
+    "./*": "./src/*/index.ts",
```

Comment:
If possible, avoid wildcards

Reply:
I thought the idea was that the build script would convert them to `./CodeHighlighter` using a strict `*` instead of `**` that package.json uses. Happy to hardcode each export if that's what you prefer

Reply:
Current build process doesn't do that. The plan is to add it in future. Till then, for new packages, it'd be ideal to be explicit with the exports unless explicitly required.